infrastructure/audit/template.yaml

```diff
@@ -171,6 +171,38 @@ Resources:
       AliasName: !Sub alias/${AWS::StackName}/${Environment}/event-replay-sns-kms-key
       TargetKeyId: !Ref EventReplaySnsKmsKey
 
+  ReplayFromAthenaEcrKmsKey:
```

Comment:
most kms keys are stored in https://github.com/govuk-one-login/txma-infra/blob/main/infrastructure/core/template.yaml. if we store it here, it’ll put the keys in all our aws accounts so if future, if we need to use containers in other accounts, the keys will already be there. it'll also save us from having to migrate it over later

Reply:
> According to https://govukverify.atlassian.net/wiki/spaces/PLAT/pages/3107258369/How+to+deploy+a+container+to+Fargate+with+secure+pipelines#Step-3%3A-Create-a-repository-in-ECR we should have a separate repository per image, so I included the name of the repository/image in the name of the kay - @nacnudus 

ignore this comment